concourse/tasks/remove-healthcheck-db.yml

```diff
@@ -13,11 +13,17 @@ run:
     - |
       ./paas-cf/concourse/scripts/import_bosh_ca.sh
       . ./config/config.sh
-      if curl -I -f $API_ENDPOINT/info; then
-        echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
-        cf target -o testers -s healthcheck
-        cf delete healthcheck -f -r
-        if cf services | grep -q healthcheck-db; then
-          cf delete-service healthcheck-db -f
-        fi
+      if ! curl -I -f $API_ENDPOINT/info; then
+        echo "CF API unavailable. Skipping..."
+        exit 0
+      fi
+      echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+      if ! cf org testers > /dev/null; then
```

Comment:
Well, I see it differently. API might not be available for other reasons ( networking issue, broken deployment as this is dev ) and we should continue deleting the deployment as this is ultimate task of this pipeline. It's only there as we want to reduce costs by not keeping VMs during night and that's the current behaviour of
```
      if ! curl -I -f $API_ENDPOINT/info; then
        echo "CF API unavailable. Skipping..."
        exit 0
      fi
```
To continue if API isn't there.

Apart from that there is an issue with using `curl -f`.
```
       -f, --fail
              (HTTP)  Fail  silently (no output at all) on server errors. This is mostly done to better enable scripts etc to better deal with failed attempts. In
              normal cases when an HTTP server fails to deliver a document, it returns an HTML document stating so (which often also describes why and more). This
              flag will prevent curl from outputting that and return error 22.

              This  method  is  not  fail-safe  and  there  are occasions where non-successful response codes will slip through, especially when authentication is
              involved (response codes 401 and 407).
```